python/full_bundle.py

```diff
@@ -171,10 +183,9 @@ def create(self) -> (int, "FullBundle"):
 
         log.info("Completed creating service-level diagnostics.")
 
-        # Find and dispatch to the appropriate BaseTechBundle.
```

Comment:
Please keep the above comment. It is the preamble to the comment below.

Reply:
changed